README.md

```diff
@@ -109,6 +109,50 @@ Warning: when checking an operation such as `getAll()` or `clear()`,
 keys, make sure to also blacklist the `""` key as well!
 
 
+## Decrypted preferences
+
+By default, devices with Android N+ come with file-based encryption, which prevents
+RemotePreferences from accessing them before the first unlock after reboot. If preferences need to be
+accessed before the first unlock, the following modifications are needed.
+
+1\. Call to the super constructor in the RemotePreferenceProvider
+```Java
+public class MyPreferenceProvider extends RemotePreferenceProvider {
+    public MyPreferenceProvider() {
+        super("com.example.app.preferences", new RemotePreferenceFile[]
+                {new RemotePreferenceFile("main_prefs", true)});
+    }
+}
+```
+The `true` will cause the provider to use context.createDeviceProtectedStorageContext()
+for obtaining the preferences.
+
+2\. To make this change effective in your activity,
```

Comment:
I don't think this section is relevant to RemotePreferences, if someone is using device encrypted storage they should already have done this elsewhere in their application. But it's your call, I'm not very familiar with the new API.